src/specs/api.yaml

```diff
@@ -101,23 +101,66 @@ paths:
       tags:
         - work
     parameters: []
-  "/experiments/{experimentId}":
+  /pipelineResults:
+    post:
+      operationId: receivePipelineResponse
+      x-eov-operation-id: 'pipelines#response'
+      x-eov-operation-handler: routes/pipelines
+      requestBody:
+        description: The data sent by AWS SNS.
```

Comment:
I would say `The results from the execution of a pipeline task, sent via SNS`

Reply:
Fixed!